code/modules/mob/living/carbon/alien/diona/diona_nymph.dm

```diff
@@ -103,8 +113,7 @@
 		holder_type = null
 
 	species = all_species[new_species]
-	if(species.language)
-		add_language(species.default_language)
+	add_language(species.default_language)
```

Comment:
A proc runtiming halts the entire execution stack. In this case, stops all procs that are supposed to get called after `set_species()` to be stopped, and carries the risk of producing fantastical errors for the user. A potentially fun example is the lack of sprites being generated, or what have you (depending on how far this specific call is).

During runtime, when an emergency hotfix takes nominally 24 hours to be produced and pushed live, such apparent errors should be avoided at all costs, in high level procs. Instead, graceful error catching should be implemented, as to minimize the effect on the end user.

In this specific case, the admins can quite easily keep adding the language to the Diona who spawn and adminhelp for a bit. The alternative is that the nymphs are unplayable until a patch is pushed.

So please, since there was a sanity check, simply adjust it and reimplement it.